hoomd/ForceCompute.h

```diff
@@ -247,52 +237,53 @@ class PYBIND11_EXPORT ForceCompute : public Compute
  *
  * */
 template<class Output>
-class PYBIND11_EXPORT LocalForceComputeData : public LocalDataAccess<Output, ForceCompute>
+class PYBIND11_EXPORT LocalForceComputeData : public PerGroupLocalDataAccess<Output, ForceCompute>
     {
     public:
-    LocalForceComputeData(ForceCompute& data)
-        : LocalDataAccess<Output, ForceCompute>(data), m_force_handle(), m_torque_handle(),
-          m_virial_handle(), m_virial_pitch(data.getVirialArray().getPitch()),
-          m_buffers_writeable(data.getBuffersWriteable())
+    LocalForceComputeData(ForceCompute& data, ssize_t N, ssize_t N_ghost, ssize_t N_global)
```

Comment:
Why a signed `ssize_t`? `GPUArray` and `GlobalArray` return a `size_t` number of elements.

Reply:
There was a narrowing conversion warning somewhere. I forget where now that I am two weeks removed, but I can check.